src/mime/LICENSE

```diff
@@ -0,0 +1,25 @@
+NOTE: The mime database in this folder (`standard.json`) was taken from
+https://github.com/broofa/node-mime#readme, which is licensed with the following license.
+The code written to read this file was written by us.
```

Comment:
Note to reviewer: I did this to reduce the package size. The `mime` package, even though it doesn't have any dependencies, is fairly large because of the additional more full-featured database that it includes. Since it's those additional (rarely-used) types are unlikely to be needed, and we're trying to keep our `@silvermine/lambda-express` package size small, I just imported the "lite" database and wrote my own wrapper for it. The MIT licensing made this acceptable to do.